flowpm/mesh_composite.py

```diff
@@ -0,0 +1,120 @@
+""" Composite FastPM elements, implemented in Mesh Tensorflow"""
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+import numpy as np
+import tensorflow.compat.v1 as tf
+tf.disable_v2_behavior()
+import tensorflow_probability as tfp
+
+import mesh_tensorflow as mtf
+
+from . import mesh_ops
+from . import mesh_utils
+from . import mesh_kernels
+
+
+
+def fr_to_hr(field, hr_shape, halo_size, splittables, mesh):
```

Comment:
Hum..... so what does `fr` mean? we might want a better name for this function

Reply:
We also need some documentation of the function, checkout how I documented functions over there: https://github.com/modichirag/flowpm/blob/dcddecd72c803f3aeadc6ad6ed548d507e6f39d7/flowpm/tfpm.py#L23